scripts/docker-run-abi-test.sh

```diff
@@ -7,8 +7,8 @@
 #
 set -e
 
-PG_MAJOR=${PG_MAJOR:-10}
-PG_MINOR_COMPILE=${PG_MINOR_COMPILE:-2}
+PG_MAJOR=${PG_MAJOR:-11}
+PG_MINOR_COMPILE=${PG_MINOR_COMPILE:-0}
```

Comment:
Why 11.0 when it was 10.2? I would expect that there are no ABI changes in PG11 compared to 11.0, but then I would expect the same for PG10, so could you check if there is any reason that we should use a later version of PG11? If you don't find an obvious reason to use a later minor version, we should go with 11.0: we will discover any problems soon enough anyway.

Reply:
@mkindahl Because 10.2 is the lowest PG 10 version, TimescaleDB was supporting.

Reply:
Note, these are the default values, which are not used, since they are set before calling the script.